changelog/fragments/rename-generate-defaults.yaml

```diff
@@ -0,0 +1,14 @@
+entries:
+  - description: >
+      `generate <bundle|packagemanifests>` will no longer write kustomize manifests bases and kustomization.yaml;
+      instead, `generate kustomize manifests` will do so
+    kind: change
+  - description: >
+      `--kustomize` has been removed from `generate <bundle|packagemanifests>`. Use `generate kustomize manifests`
+      to write manifests bases and kustomization.yaml instead
+    kind: removal
+  - description: >
+      `bundle` will write manifests/metadata to the top-level `bundle/` directory,
+      and `packagemanifests` will write manifests and the package manifest to
+      the top-level `packagemanifests/` directory
+    kind: change
```

Comment:
Do we need to document these changes to the new CLI cmds `generate bundle/packagemanifests` when the new CLI wasn't even exposed in `v0.18.1`?
So users probably don't have a reference for this change of `generate --kustomize` ==> `generate kustomize manifests` in the new CLI.

Maybe we just document that we now have `generate bundle/packagemanifests` cmds as part of the new CLI.
Which will just add to the CHANGELOG for the new CLI https://github.com/operator-framework/operator-sdk/pull/3254

Reply:
You're right, this shouldn't have a fragment. There's no documentation/fragments discussing where stuff is generated either.